### PR TITLE
Update to deps.edn and clearing some test

### PR DESCRIPTION
**Pull request by davidpham87**
Hello,

I desired to use your library for one of my project, and I could not use it with `deps.edn`, so I quickly changed it. And since your were nice enough to write test, I also cleared some of them so that all test passes (I basically hacked the date time regex).

Thanks a lot for your work.

### CODE REVIEW

**Comment by bnbeckwith** [comment]
@davidpham87, thank you for the PR.  I'll need to update my knowledge on how =deps.edn= works to fully approve the request.  I'll add some comments to your PR soon and we can go from there.

File: README.org [hunk 1]
```diff
@@ -3,23 +3,33 @@
   A Clojure library designed to parse mode files into clojure data
   structures.
 
+** Status
+   Usable, but not perfect, parse the [[test/orgmode/test.org][test]] file to see some of the issues.
+
 ** Installation
 
-   If you are using =lein=, then just add it as a dependency.
+   Using =deps.edn=
```
Comment (bnbeckwith) [inline README.org]: Can we have both =lein= and =deps.edn=?

File: README.org [hunk 2]
```diff
@@ -3,23 +3,33 @@
...
 
 #+BEGIN_SRC clojure
-  [bnbeckwith.com/orgmode "0.7.2"]
+  {:deps {orgmode {:git/url "https://github.com/davidpham87/orgmode"
```
Comment (bnbeckwith) [inline README.org]: Should this be a link to my repository?

File: src/orgmode/inline.clj [hunk 3]
```diff
@@ -40,25 +40,28 @@
 
 ;; ### Timestamp Elements
 
+;; hack to make it work with ranges
```
Comment (bnbeckwith) [inline src/orgmode/inline.clj:43]: Pedantic: Is it a _hack_ or a _fix_?

File: src/orgmode/inline.clj [hunk 4]
```diff
@@ -147,3 +151,12 @@
       (make-elem (fmt-re "_")   (fmt-create :underline))
       (make-elem (fmt-re "=")   (fmt-create :verbatim))
       (make-elem (fmt-re "~")   (fmt-create :code))))
+
+
+(comment
```
Comment (bnbeckwith) [inline src/orgmode/inline.clj]: I think that we can remove this comment and turn it into a test?

**Review by bnbeckwith — CHANGES_REQUESTED** [review]
Thanks for the PR!

I think that your documentation commits can be squashed together. Otherwise, I left a few comments inline so that I can understand all of the parts.

File: src/orgmode/inline.clj (hunk 3 above)
Comment (davidpham87) [reply src/orgmode/inline.clj:43]: I don't really know, honestly :-). It is hack in the sense that I don't why you get an intermediate grouping (my regexfu is limited), this is why it is hack. Empirically, it provides the requested results though.

File: README.org (hunk 2 above)
Comment (davidpham87) [reply README.org]: Yes it should be your repository, once we get the merge.

File: README.org (hunk 1 above)
Comment (davidpham87) [reply README.org]: Probably yes, I can add it back. I could not find where your jar was located in maven/clojars though.

File: src/orgmode/inline.clj (hunk 4 above)
Comment (davidpham87) [reply src/orgmode/inline.clj]: Okay.

**Comment by davidpham87** [comment]
Voilà! I hope the change answers your requests.

**Review by bnbeckwith — APPROVED** [review]
Looks good.  Thanks!

**Pull request merged**